server/terraform/README.md

```diff
@@ -25,22 +25,54 @@ This uses the `terraform@who-terraform-admin.iam.gserviceaccount.com` service ac
 - [Terraform adding credentials](https://www.terraform.io/docs/providers/google/guides/getting_started.html#adding-credentials)
 - [Securely handling keys](https://cloud.google.com/iam/docs/understanding-service-accounts?_ga=2.87249435.-2051693357.1581897767#managing_service_account_keys)
 
-### Terraform service account
+### Terraform Service Account
 
-Only needed for terraform service account setup. The account must have these permissions:
+Service accounts are used to access the cloud projects. Development account
+needs to create projects and therefore needs these permissions:
```

Comment:
Oh interesting... so you're using terraform to create a new project from scratch. Who runs this? Is it done by a project admin (e.g.  you) to create a project for use by a new developer as their scratch space?

Note: I am trying to create my own setup, and don't see either of these roles as options when I create a service account. There is project owner and project billing manager. Presumably this means that the project you're looking it is somehow a level 'up' in some hierarchy of project ownership

Reply:
I think you might be creating the service account at the project rather than organization level. I've updated the PR about this but it probably needs more attention again.